guides/common/modules/proc_updating-disconnected-server.adoc

```diff
@@ -0,0 +1,163 @@
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel8/8/x86_64/baseos/os
+enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
+
+[{RepoRHEL8AppStream}]
+name=Red Hat Enterprise Linux 8 for x86_64 - AppStream (RPMs)
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/rhel8/8/x86_64/appstream/os
+enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
+
+[{RepoRHEL8ServerSatelliteServerProductVersion}]
+name={ProjectName} {ProjectVersion} for RHEL 8 RPMs x86_64
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/layered/rhel8/x86_64/satellite/{ProjectVersion}/os
+enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+
+[{RepoRHEL8ServerSatelliteMaintenanceProductVersion}]
+name={ProjectName} Maintenance {ProjectVersion} for RHEL 8 RPMs x86_64
+baseurl=https://{foreman-example-com}/pulp/content/My_Organization/Library/content/dist/layered/rhel8/x86_64/sat-maintenance/{ProjectVersion}/os
+enabled=1
+sslclientcert = /etc/pki/katello/certs/org-debug-cert.pem
+sslclientkey = /etc/pki/katello/certs/org-debug-cert.pem
+sslcacert = /etc/pki/katello/certs/katello-server-ca.crt
+sslverify = 1
+----
++
+. In the configuration file, complete the following steps:
+* For the `sslclientcert` and `sslclientkey` options, replace `/etc/pki/katello/certs/org-debug-cert.pem` with the location of the downloaded organization debug certificate.
+* For the `{foreman-example-com}` option, specify the correct FQDN for your deployment.
+* For the `baseurl` option, replace `My_Organization` with the correct organization label.
+
+. Obtain the organization label:
++
+----
+# hammer organization list
+----
+
+. Enter the `reposync` command:
++
+[options="nowrap" subs="attributes"]
+----
+# reposync --delete --download-metadata -p ~/{project-context}-repos -n \
+ --disableplugin=foreman-protector \
+ --repoid {RepoRHEL8BaseOS} \
+ --repoid {RepoRHEL8AppStream} \
+ --repoid {RepoRHEL8ServerSatelliteServerProductVersion} \
+ --repoid {RepoRHEL8ServerSatelliteMaintenanceProductVersion}
+----
++
+This downloads the contents of the repositories from the connected {ProjectServer} and stores them in the directory `~/{Project}-repos`.
++
+. Verify that the RPMs have been downloaded and the repository data directory is generated in each of the sub-directories of `~/{Project}-repos`.
+. Archive the contents of the directory:
++
+[options="nowrap" subs="attributes"]
+----
+# cd ~
+# tar czf {Project}-repos.tgz {Project}-repos
```

Comment:
I'd use `-C` to change directory.
```suggestion
# tar czf {Project}-repos.tgz -C ~ {Project}-repos
```

Or use the self documenting long option `--directory`.